.ci.yaml

```diff
@@ -98,7 +98,8 @@ platform_properties:
             {"name":"openjdk","path":"java"},
             {"name":"pub_cache","path":".pub-cache"},
             {"name":"xcode_binary","path":"xcode_binary"},
-            {"name":"osx_sdk","path":"osx_sdk"}
+            {"name":"osx_sdk","path":"osx_sdk"},
+            {"name":"xcode_runtime_ios-15-0","path":"xcode_runtime_ios-15-0"}
```

Comment:
Did I do this correctly?
https://flutter-review.googlesource.com/c/recipes/+/24944/6/recipe_modules/osx_sdk/api.py

Reply:
/cc @godofredoc for the cache set up.

Reply:
No, validator didn't like that:
```
Mac_ios animation_with_microtasks_perf_ios__timeline_summary / caches #8): name: "xcode_runtime_ios-15-0" does not match "^[a-z0-9_]+$"
```

Reply:
I think it's something like `runtime_versions: ["ios-15-0"]` but I'm not sure where that property goes in the flutter framework, in an engine `led` run I did `led edit -p '$flutter/osx_sdk={"sdk_version": "13a233", "runtime_versions": ["ios-13-0", "ios-15-0"] }'`.